templates/20-04/aws.html

```diff
@@ -0,0 +1,232 @@
+              </li>
+              <li class="p-list__item has-bullet">
+                Key infrastructure components
+              </li>
+              <li class="p-list__item has-bullet">
+                Over 25,000 open-source applications until 2030, adding 5 years of support. It includes
+              </li>
+              <li class="p-list__item has-bullet">
+                24-hour critical vulnerability fixes, Landscape, kernel livepatching, hardening scripts based on CIS and DISA-STIG standards, and FIPS 140 compliance, ensuring your infrastructure meets U.S. government security requirements for cryptographic modules.
+              </li>
+            </ul>
+            <p>
+              Ubuntu Pro 20.04 is available with pay-as-you-go metered pricing on AWS, with charges automatically billed to your monthly AWS bill for easy procurement. Additionally, Ubuntu Pro subscriptions can count towards your PPA/EDP spend commitments. 
+            </p>
+          </div>
+        </li>
+      </ol>
+    </div>
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h3>Add-ons</h3>
+      </div>
+      <div class="col">
+        <h4 class="p-heading--5 u-no-margin--bottom">Enterprise-grade phone & ticket support</h4>
+        <p>
+          Build confidently with 24/7/365 phone and ticket support, ensuring fast resolution for issues across 25,000+ Ubuntu packages, backed by 20 years of expertise in building and sustaining open source software. Learn more <a href="/aws/support">here</a>.
+        </p>
+        <hr class="p-rule--muted" />
+        <h4 class="p-heading--5 u-no-margin--bottom">Legacy support</h4>
+        <p>Add on Legacy Support to Ubuntu Pro and expand security maintenance (coverage through 2032). Learn more <a href="https://canonical.com/blog/canonical-expands-long-term-support-to-12-years-starting-with-ubuntu-14-04-lts">here</a>.</p>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="u-fixed-width">
+      {{ image(url="https://assets.ubuntu.com/v1/c8af1245-server-room.png",
+        alt="",
+        width="2464",
+        height="1028",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </section>
+
+  {% call(slot) vf_quote_wrapper(
+    title_text="What customers say about Ubuntu Pro",
+    quote_size="medium",
+    quote_text="ESM literally saved our lives. It's allowing us to upgrade from 14.04 LTS at our own pace. It's taken the pressure off, and it also means we can tackle the Ubuntu upgrades at the same time as we roll out the new version of our platform.",
+    citation_source_name_text="Zivago Lee",
+    citation_source_title_text="Director of DevOps Engineering",
+    citation_source_organisation_text="Interana",
+    is_shallow=True
+    ) -%}
+
+    {%- if slot == 'signpost_image' -%}
+      {{ image(url="https://assets.ubuntu.com/v1/fe89136b-updated-interana.png",
+            alt="",
```

Comment:
```suggestion
            alt="Interana",
```
For company logos, we usually provide alt text as the logo itself.